modules/openy_features/openy_node/modules/openy_node_blog/src/BlogAlertBuilder.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\openy_node_blog;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\openy_node_alert\Service\AlertBuilderInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Provides an alert builder for blog post pages.
+ */
+class BlogAlertBuilder implements AlertBuilderInterface {
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * Constructs the BlogAlertBuilder.
+   *
+   * @param EntityTypeManagerInterface $entity_type_manager
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->nodeStorage = $entity_type_manager->getStorage('node');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(EntityInterface $node) {
+    return $node->bundle() == 'blog' && !$node->field_blog_location->isEmpty();
```

Comment:
Sometimes it uses ===, sometimes == for node bundle checks -> please change to === in all the cases.